From: Ondrej Jirman <megi@xff.cz>
Date: Fri, 2 Dec 2022 22:55:59 +0100
Subject: bluetooth: h5: Don't re-initialize rtl8723cs on resume

The power is kept across during sleep, and this is not necessary.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 drivers/bluetooth/hci_h5.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/bluetooth/hci_h5.c b/drivers/bluetooth/hci_h5.c
index 0ff9d84..5d1319e 100644
--- a/drivers/bluetooth/hci_h5.c
+++ b/drivers/bluetooth/hci_h5.c
@@ -1103,7 +1103,7 @@ static const struct of_device_id rtl_bluetooth_of_match[] = {
 	{ .compatible = "realtek,rtl8723ds-bt",
 	  .data = (const void *)&h5_data_rtl8723bs },
 	{ .compatible = "realtek,rtl8723cs-bt",
-	  .data = (const void *)&h5_data_rtl8723bs },
+	  .data = (const void *)&h5_data_rtl8822cs },
 #endif
 	{ },
 };
